src/zjs_sensor_frdm.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Rename this file from frdm to k64f, that's what we've been using elsewhere as a short abbreviation.